src/main/java/org/jabref/logic/util/URLUtil.java

```diff
@@ -7,13 +7,20 @@
 import java.net.URLDecoder;
 import java.nio.charset.StandardCharsets;
 import java.util.Objects;
+import java.util.regex.Pattern;
 
 /**
  * URL utilities for URLs in the JabRef logic.
  * <p>
  * For GUI-oriented URL utilities see {@link org.jabref.gui.fieldeditors.URLUtil}.
  */
 public class URLUtil {
+    private static final String URL_REGEX = "(?i)\\b((?:https?|ftp)://[^\\s]+)";
+    /**
+     * Pattern match a string containing a URL with a protocol
```

Comment:
```suggestion
     * Pattern matches a string containing a URL with a protocol
```

Please a add a empty line before the JavaDoc, too. Otherwise, it is too close to the other variable